test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -45,6 +45,8 @@ var (
 			"scheduler_scheduling_algorithm_priority_evaluation_seconds",
 			"scheduler_binding_duration_seconds",
 			"scheduler_e2e_scheduling_duration_seconds",
+			"scheduler_scheduling_algorithm_preemption_evaluation_seconds",
```

Comment:
Is this metric from scheduler metrics? If so, I can only find `scheduling_algorithm_preemption_evaluation_seconds`.

Reply:
yes, the prefix is added by default to all metrics reported by the scheduler (see older metrics above).

Reply:
nvm, the `scheduler_` prefix is added by the Kubernetes prometheus library according to defined `Subsystem`:

https://github.com/kubernetes/kubernetes/blob/bbaa76c8f514eada3f3c30bd20d8b59fd623711e/pkg/scheduler/metrics/metrics.go#L169